src/base/utils/misc.h

```diff
@@ -30,14 +30,13 @@
 #define UTILS_MISC_H
 
 #include <QtGlobal>
+#include <QString>
```

Comment:
wrong place, move it to the same line before.

Reply:
done?

Reply:
no

Reply:
place it where the `QString` declaration was?

Reply:
```c++
#include <QtGlobal>

#ifdef Q_OS_WIN
#include <memory>
#include <Windows.h>
#endif

#include <QString>
```

Reply:
ok?

Reply:
not quite, you need to squash commits.

Reply:
ok?

Reply:
still no. in commit message, what do you mean Window? or do you mean Windows?
also Compilation needs to start with lowercase.

Reply:
XD sorry. now?

Reply:
now starts the real review...